src/Tickets/Flexible_Tickets/Custom_Tables/Ticket_Groups.php

```diff
@@ -71,14 +71,19 @@ protected function get_definition() {
 	 * {@inheritdoc}
 	 *
 	 * @since 5.24.1 Handle hydrating new columns from `data` JSON for Ticket Presets use, if needed.
+	 * @since TBD    Handle MySQL compatibility fix for TEXT column DEFAULT value removal.
 	 */
 	protected function after_update( array $results = [] ) {
 		$results = parent::after_update( $results );
 
 		// Run version-specific migrations.
-		if ( self::SCHEMA_VERSION === '1.1.0' ) {
-			$results = $this->migrate_to_1_1_0( $results );
-		}
+		switch ( self::SCHEMA_VERSION ) {
```

Comment:
Just curious: what if someone updates from `1.0.0` directly to a later version, say `1.3.0`? Do these still need to run?

When updating from `1.0.0` to `1.2.0`, doe the `1.1.0` logic still need to run?

Reply:
Hm, good point - not in this case, but potentially. 🤔